content/common/contributing-docs.md

```diff
@@ -149,6 +149,66 @@ With all Node dependencies installed, you can now check for broken links and als
 $ npm test
 ```
 
+### 6.2 Checking Your Content - Using Bartholomew's CLI
+
+The Bartholomew Command Line Interface (CLI) Tool is called `bart`. The `bart` CLI is a tool that simplifies working with Bartholomew projects (by now you probably already know that [Bartholomew](https://www.fermyon.com/blog/introducing-bartholomew) is our in-house WebAssembly (Wasm) content management system (CMS) that powers [our official Website](https://www.fermyon.com/)). And this (our official documentation) site. The `bart` CLI is handy to ensure quality assurance of new and existing content. Installing the CLI is a cinch, so please go ahead and use it when contributing.
```

Comment:
TIL "cinch".
(https://www.merriam-webster.com/dictionary/cinch)

Reply:
I have only traditionally know the word "cinch" as a noun which represents "a thing done with ease". TIL (thanks to your link) it is also a "a strap that holds a saddle on a horse". Gosh the English language is interesting right?